content/blog/2024-12-03-tmil-11/index.md

```diff
@@ -0,0 +1,158 @@
+
+Linebender, among other things, believes in helping build solid foundations for people building user interfaces and working in graphics, in the Rust programming language (and on GPUs).
+Upon seeing a problem, we prefer to produce a focused crate that solves that problem and that can be used by others within the Rust ecosystem.
+Color is one such area.
+It is much harder than it may appear at first, with many complicating factors like what should be done by a basic color struct, what color spaces to support, and whether or not to support color formats used in print (like CMYK).
+
+We looked around at a wide variety of crates in various stages of maintenance and development, their feature sets, the "weight" of those crates, and how closely they fit in with what we would need in Vello, tiny-skia, resvg, svgtypes, Masonry, and Xilem.
+We decided to do a crate based on the [CSS Color Module Level 4] draft specification, with support for a limited number of additional color spaces.
+
+We drew a line in the sand and released a preview of our crate as [color 0.1.0][] on November 21, 2024.
+The documentation lays out more of the scope, goals, and main types provided by the crate.
+
+Since then, we have begun the work of making Peniko and Vello use the Color crate and improving it to support that usage.
+Early next year, we will use it within svgtypes, resvg, and tiny-skia.
+
+We will be releasing a 0.2.0 release within the next month and encourage people to give it a try.
+The 0.2.0 release will include some additional (non-CSS 4) color spaces like ACEScg, OkHsl, and OkHsv as well as things that we found useful when using it in Peniko and Vello.
+
+Many thanks to [Brendan Zabarauskas][] for working with us to transfer his `color` crate name for our use.
+His original crate was published as 0.0.1 on November 21, 2014, motivating us ten years later to ship our 0.1.0 on November 21, 2024.
+
+We're very excited about the future of this crate and look forward to talking to other people within the Rust ecosystem about using it.
+
+## Kurbo
+
+Kurbo has seen some minor updates within the last month and we're planning to publish them in a minor patch release within the next month.
+
+* [kurbo#390][] Reduce number of operations in `Triangle::circumscribed_circle`.
+* [kurbo#399][] Implement `Sum` for `Vec2`.
+
+Internally, we're talking about the future direction of Kurbo and how we can improve upon it as a foundational crate for curves, shapes, and math.
+Hopefully we'll be writing about that in more detail within the next couple of months.
+
+## Peniko
+
+Peniko will have a big release within the next month, once the new release of Color is out, that will remove the old `peniko::Color` type and replace it with the new create.
+This release will be the start of a path towards wider color gamut support throughout the Linebender ecosystem.
```

Comment:
Less about the future, more about November.

```suggestion
We've been working towards a big release, which will remove the old `peniko::Color` type and use the new `color` crate instead.
This eventual release will be the start of a path towards wider color gamut support throughout the Linebender ecosystem.
```

Reply:
I agree with the first paragraph; I think "eventual" is probably going too far the other way.

Reply:
Edited your suggestion to remove "eventual" ... it isn't eventual. It is happening within the next 2 weeks, probably 1 week.